cpp/density/SphereVoxelization.h

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2010-2019 The Regents of the University of Michigan
```

Comment:
Make sure all the copyrights are updated (see #602).
```suggestion
// Copyright (c) 2010-2020 The Regents of the University of Michigan
```